packages/design-system/src/components/Dropdown/Dropdown.stories.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import Dropdown from './Dropdown';
+import Select from './Select';
+
+export default {
+  title: 'Components/Dropdown',
+  component: Dropdown,
+  argTypes: {
+    children: { control: false },
+    options: { control: false },
+  },
+  parameters: {
+    backgrounds: {
+      default: 'light',
+      values: [
+        {
+          name: 'light',
+          value: '#fff',
+        },
+        {
+          name: 'Hcgov dark',
+          value: '#112e51',
+        },
+        {
+          name: 'Mgov dark',
+          value: '#146a5d',
+        },
+      ],
+    },
+  },
+  subcomponents: { Select },
+};
+
+const dropdownOptions = [
+  { label: '- Select an option -', value: '' },
+  { label: 'Option 1', value: '1' },
+  { label: 'Option 2', value: '2' },
+  { label: 'Option 3', value: '3' },
+  { label: 'Option 4', value: '4' },
+  { label: 'Option 5', value: '5' },
+  { label: 'Option 6', value: '6' },
+  { label: 'Option 7', value: '7' },
+  { label: 'Option 8', value: '8' },
+];
+
+const optGroupData = (
+  <>
+    <optgroup label="Option group">
+      <option value="1-1">Option 1</option>
+      <option value="1-2">Option 2</option>
+      <option value="1-3">Option 3</option>
+    </optgroup>
+    <optgroup label="More option groups">
+      <option value="2-1">Option 4</option>
+      <option value="2-2">Option 5</option>
+      <option value="2-3">Option 6</option>
+    </optgroup>
+  </>
+);
+
+const Template = (args) => <Dropdown {...args} />;
+
+export const DefaultDropdown = Template.bind({});
+DefaultDropdown.args = {
+  options: dropdownOptions,
+  label: 'Dropdown Example',
```

Comment:
Not to be a stickler, but this example's using title case when our docs don't ("Dropdown example"). Not sure if this is something we enforce as a design system.